src/main/java/io/parallec/core/commander/workflow/ssh/SshProvider.java

```diff
@@ -151,6 +169,53 @@ public Session startSshSessionAndObtainSession() {
     }
 
     /**
+     * Start a Session when using jump host (port forward).
+     *
+     * @return the session
+     */
+    public Session startSshForwardSession() {
+        SshMeta jumpMeta = sshMeta.getSshJumpHostMeta();
+        Session session = null;
+        Session forwarded = null;
+        JSch jsch = new JSch();
+        try {
+            setJSchByMeta(jsch, sshMeta.getSshJumpHostMeta());
+            session = jsch.getSession(jumpMeta.getUserName(), sshMeta.getSshJumpHost(),
+                    jumpMeta.getSshPort());
+
+            if (jumpMeta.getSshLoginType() == SshLoginType.PASSWORD) {
+                session.setPassword(jumpMeta.getPassword());
+            }
+
+            session.setConfig("StrictHostKeyChecking", "no");
+
+            session.connect();
+
+            logger.debug("The session has been established to {}@{}", jumpMeta.getUserName(),
+                    sshMeta.getSshJumpHost());
+
+            jsch = setJSchByMeta(jsch, sshMeta);
```

Comment:
With lIne 177 and line 197 set twice, I understand the 2 metadata with the jump host info and this time the original metadata info being set into the jsch object.  however its readability could be improved. Suggest not to use the Class sshMeta to wrap the 3 attributes needed for the jump host.

Also seem with the `setJSchByMeta` function, we do not need the "=" assignment?   like 182 did not use the assignment